scripts/deploy_oak_functions_api

```diff
@@ -53,6 +53,11 @@ sed \
 readonly CONFIG_ID='weather-lookup-config'
 readonly API_ID='weather-lookup-api'
 
+# First delete the existing api-config and gateway
+# gcloud api-gateway gateways delete "${GATEWAY_ID}" --location="${GCP_REGION}"
+# gcloud api-gateway api-configs delete "${CONFIG_ID}" --api="${API_ID}" --project="${GCP_PROJECT_ID}"
+
```

Comment:
I think an extra step is needed to delete the API as well with `gcloud api-gateways apis delete ...` - the API is automatically created when creating the initial config, but not automatically deleted.

Also, you probably need to disable the API before being able to delete is (I think `gcloud services disable "${MANAGED_SERVICE}"` should do it.